nanofi/examples/monitor_directory.c

```diff
@@ -75,11 +77,11 @@ int main(int argc, char **argv) {
 
   port.port_id = portStr;
 
-  C2_Server server;
-  server.url = argv[4];
-  server.ack_url = argv[5];
-  server.identifier = "monitor_directory";
-  server.type = REST;
+  // C2_Server server;
```

Comment:
This is probably leftover commented-out code.

Reply:
Removed commented code in ee40d7658735b8ebe61f88956dfa6581da3cb863